packages/core/src/enqueue_links/shared.ts

```diff
@@ -79,7 +159,8 @@ export function createRequestOptions(sources: (string | Record<string, unknown>)
             } catch (err) {
                 return false;
             }
-        });
+        })
+        .map((requestOptions) => new Request(requestOptions) as RequestOptions);
```

Comment:
@B4nan the reason for this is that request creates `userData` and proper request properties, while creating an object does not do it. So I thought instead of creating `userData` explicitly I would just create a request. On the other hand - you are right that there are requestOptions method and request creation method. So I guess let's create the `userData` explicitly for requestOptions - will update it

Reply:
Updated, but it still feels a bit weird with a check for userData existence and creating an empty object if it's not there.